plugin/session_buffer.py

```diff
@@ -49,9 +49,11 @@ def update(self, version: int, changes: Iterable[sublime.TextChange]) -> None:
 
 class DiagnosticSeverityData:
 
-    __slots__ = ('regions', 'regions_with_tag', 'annotations', 'scope', 'icon')
+    __slots__ = ('region_diagnostics', 'tag_diagnostics', 'regions', 'regions_with_tag', 'annotations', 'scope', 'icon')
 
     def __init__(self, severity: int) -> None:
+        self.region_diagnostics = []  # type: List[Diagnostic]
+        self.tag_diagnostics = []  # type: List[Diagnostic]
```

Comment:
This change is a bit of a hack I guess since it stores whole diagnostics in a class whose name suggest that it should be only "diagnostic severity data"...